### PR TITLE
feat(linter): Implement eslint(getter-return)

### PR DESCRIPTION
**Pull request by Maneren**

### CODE REVIEW

**Review by Boshen — CHANGES_REQUESTED** [review]

File: crates/oxc_linter/src/rules/getter_return.rs [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+            let ObjectPropertyKind::ObjectProperty(property) = property else { continue };
+            if !property.key.static_name().is_some_and(|name| name == "get") {
+                continue;
+            }
+
+            match &property.value {
+                Expression::FunctionExpression(function) => {
+                    let Some(body) = &function.body else { continue };
+                    if !self.is_correct_getter(body) {
+                        let span = Span {
```
Comment (Boshen) [inline crates/oxc_linter/src/rules/getter_return.rs:70]: nit: use `Span::new`

**Review by Boshen — CHANGES_REQUESTED** [review]

**Comment by Boshen** [comment]
`cargo test && cargo insta accept`

**Comment by Maneren** [comment]
Yh, I went out and am only on a phone. Will fix it when I get back home in a few hours.

**Comment by Boshen** [comment]
I would not have implemented better than this, thank you so much!

**Pull request merged**